config/routes.rb

```diff
@@ -9,8 +9,19 @@
   get 'rentals/:id/invoice', to: 'rentals#invoice', as: 'rental_invoice'
   get 'rentals/safety_pdf'
   get 'rentals/training_pdf'
-  resources :rentals
 
+  #---Help Forms-------------------------------------------------------------
+  get 'help', to: 'help#show'
```

Comment:
Curious why you're constructing your routes this way. Do you need all these individual path helpers? (`as: 'path'`)
What if you did 
``` ruby
resources :help, :none do
  collection do  
    get :schedule
    etc
  end
end
```
I think you'd have the exact same path helpers in most cases, also, and it would be a lot cleaner.

Reply:
True, will change